docs/policies/recognizing_contributors_sponsors_and_partners.rst

```diff
@@ -0,0 +1,115 @@
+Recognizing contributors, sponsors and partners
+###############################################
+
+Background information 
+======================
```

Comment:
Please take care of your headings, this is a H3 and it should be a H2. This will need checking and addressing throughout the document.

```
H1: ###
H2: ***
H3: ===
H4: ---
H5: ~~~
```